core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSearchRequestBuilder.java

```diff
@@ -35,46 +35,12 @@
 	public ConceptSearchRequestBuilder filterByActive(Boolean active) {
 		return addOption(OptionKey.ACTIVE, active);
 	}
-	
-	/**
-	 * Filters matches by their lexical terms. The exact semantics of how a term match works depends on the given code system, but usually it supports
-	 * exact, partial word and prefix matches.
-	 * 
-	 * @param term
-	 * @return
-	 */
-	public ConceptSearchRequestBuilder filterByTerm(String term) {
```

Comment:
Please keep the original methods for backward compatibility and delegate to the new `filterByTerm(TermFilter)` method.